MediaPlayer.py

```diff
@@ -325,18 +388,25 @@ def search_tracks(self, **kwargs):
 		"""
 		title = kwargs.get('title')
 		rating = kwargs.get('rating')
+		if not rating:
+			rating = False
+
```

Comment:
can be much shorter :wink: 
```suggestion
		rating = kwargs.get('rating') or False

```

Reply:
I didn't know you could do that!